archive-bibliotech.php

```diff
@@ -14,7 +14,7 @@
  *
  * @package WordPress
  * @subpackage Twenty_Twelve
- * @since Twenty Twelve 1.0
+ * @since 1.0
```

Comment:
This template should not be ascribed to WordPress, or Twenty_Twelve - it was created specifically for this theme.